src/cpp/ext/gsm/metadata_exchange.cc

```diff
@@ -42,30 +42,35 @@ namespace internal {
 
 namespace {
 
-const char kMetadataExchangeTypeKey[] = "type";
-const char kMetadataExchangePodNameKey[] = "pod_name";
-const char kMetadataExchangeContainerNameKey[] = "container_name";
-const char kMetadataExchangeNamespaceNameKey[] = "namespace_name";
-const char kMetadataExchangeClusterNameKey[] = "cluster_name";
-const char kMetadataExchangeLocationKey[] = "location";
-const char kMetadataExchangeProjectIdKey[] = "project_id";
-const char kMetadataExchangeCanonicalServiceNameKey[] =
+// The keys that will be used in the Metadata Exchange between local and remote.
+const absl::string_view kMetadataExchangeTypeKey = "type";
+const absl::string_view kMetadataExchangePodNameKey = "pod_name";
+const absl::string_view kMetadataExchangeContainerNameKey = "container_name";
+const absl::string_view kMetadataExchangeNamespaceNameKey = "namespace_name";
+const absl::string_view kMetadataExchangeClusterNameKey = "cluster_name";
+const absl::string_view kMetadataExchangeLocationKey = "location";
+const absl::string_view kMetadataExchangeProjectIdKey = "project_id";
+const absl::string_view kMetadataExchangeCanonicalServiceNameKey =
     "canonical_service_name";
-const char kPeerTypeAttribute[] = "peer_type";
-const char kPeerPodNameAttribute[] = "peer_pod_name";
-const char kPeerContainerNameAttribute[] = "peer_container_name";
-const char kPeerNamespaceNameAttribute[] = "peer_namespace_name";
-const char kPeerClusterNameAttribute[] = "peer_cluster_name";
-const char kPeerLocationAttribute[] = "peer_location";
-const char kPeerProjectIdAttribute[] = "peer_project_id";
-const char kPeerCanonicalServiceNameAttribute[] = "peer_canonical_service_name";
+// The keys that will be used for the peer attributes when recording metrics.
+const absl::string_view kPeerTypeAttribute = "gsm.remote_type";
```

Comment:
All of these strings except for canonical_service_name should be `gsm.remote_workload_*` instead of just `gsm.remote_*`.